content/22.vaccines.md

```diff
@@ -343,39 +333,22 @@ These data are retrieved from Our World in Data <!-To Do: Cite--> and plotted us
 ]({{owid_protein_subunit_map}} "Availability of protein subunit vaccines"){#fig:ps-distrib secno=1}
 
-Novavax is now preparing to apply for an EUA to the FDA [@url:https://ir.novavax.com/Novavax-Statement-on-UK-and-Mexico-Phase-3-Clinical-Trial-Participants-Considered-Fully-Vaccinated-in-the-US].<!--Still true?-->
-It has also been reported that Novavax initiated booster trials in the U.K. [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html]. 
 As of {{owid_most_recent_date}}, {{owid_protein_subunit_count}} protein subunit vaccines are being distributed in {{owid_protein_subunit_countries}} countries (Figure @fig:ps-distrib).
+<!--These include...-->
 However, the Novavax vaccine has yet to be approved in any country but is allowed for emergency use in Indonesia [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html], and according to press releases there have been applications for approval for use submitted to New Zealand, Australia, the U.K., and Canada [@url:https://ir.novavax.com/press-releases] and for emergency use listing with the WHO [@url:https://ir.novavax.com/2021-11-04-Novavax-Files-COVID-19-Vaccine-for-Emergency-Use-Listing-with-World-Health-Organization].<!--Still true?-->
-Indeed, Novavax has also promised to provide up to 1.1 billion doses to COVAX; however, there have been reports of struggle to maintain production capacity and quality, hindering its production targets [@url:https://www.politico.com/news/2021/10/19/novavax-vaccine-rush-process-global-campaign-516298].
-
-**Variants:**
-
-_Post hoc_ analysis in the phase III trial determined that the NovaVax vaccine had an efficacy of 86.3% against the Alpha variant (identified based on the presence/absence of the 69–70del polymorphism) and 96.4% against other variants [@doi:10.1056/NEJMoa2107659].
-In the second phase III trial [@doi:10.1101/2021.10.05.21264567], whole-genome sequencing was obtained from 61 of the 77 cases, and 79% of infections were identified as a VOC or VOI that had been characterized at the time of the study.
-Vaccine efficacy against cases caused by VOC, among which the Alpha variant was predominant (88.6%), was reported to be 92.6% [@doi:10.1101/2021.10.05.21264567]. 
-
-##### Virus-Like Particle
-
-<!--To Do: add info about VLP and phase 1/2 trials:
-https://www.raps.org/news-and-articles/news-articles/2020/3/covid-19-vaccine-tracker
-https://pubmed.ncbi.nlm.nih.gov/20923267/
-https://jnanobiotechnology.biomedcentral.com/articles/10.1186/s12951-021-00806-7
--->
-
-##### Recombinant Protein Vaccine
-
-<!--To Do: add, see https://www.raps.org/news-and-articles/news-articles/2020/3/covid-19-vaccine-tracker-->
+Novavax is now preparing to apply for an EUA to the FDA [@url:https://ir.novavax.com/Novavax-Statement-on-UK-and-Mexico-Phase-3-Clinical-Trial-Participants-Considered-Fully-Vaccinated-in-the-US].<!--Still true?-->
```

Comment:
It looks like they submitted the EUA in January: https://ir.novavax.com/2022-01-31-Novavax-Submits-Request-to-the-U-S-FDA-for-Emergency-Use-Authorization-of-COVID-19-Vaccine

The Wall Street Journal reported today that they expect approval in the coming weeks (original article paywalled): https://fortune.com/2022/03/07/fourth-covid-vaccine-novavax-anti-vax-protein-mrna/

Reply:
```suggestion
Novavax applied for an EUA from the FDA in early 2022 [@url:https://ir.novavax.com/2022-01-31-Novavax-Submits-Request-to-the-U-S-FDA-for-Emergency-Use-Authorization-of-COVID-19-Vaccine].
```
Thank you @agitter! New motivation to try to get this out before anything else changes 😆